yew/src/html/scope.rs

```diff
@@ -267,6 +267,24 @@ impl<COMP: Component> Scope<COMP> {
         };
         closure.into()
     }
+
+    /// Creates a `Callback` from a FnOnce which will send a batch of messages back
```

Comment:
"FnOnce" should be between ticks:
```suggestion
    /// Creates a `Callback` from a `FnOnce` which will send a batch of messages back
```

The same issue exists in `callback_once`. If you don't mind, could you update it there as well?

Reply:
Done